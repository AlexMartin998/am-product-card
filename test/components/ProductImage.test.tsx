import * as React from 'react';
import { create } from 'react-test-renderer';
import { ProductCard } from '../../src/components';

import { ProductImg } from '../../src/components/ProductImage';
import { testProduct2 } from '../data/product';

describe('first', () => {
  test('should first', () => {
    const wrapper = create(
      <ProductImg
        className="custom-class-img"
        img="./../../example"
        style={{ display: 'flex', justifyContent: 'center' }}
      />
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  test('should second', () => {
    const wrapper = create(
      <ProductCard product={testProduct2}>
        {() => <ProductImg className="custom-class-img" />}
      </ProductCard>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
