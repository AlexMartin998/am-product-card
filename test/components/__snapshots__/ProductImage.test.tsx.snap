// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first should first 1`] = `
<img
  alt="Product img"
  className="productImg custom-class-img"
  src="./../../example"
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
    }
  }
/>
`;

exports[`first should second 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product img"
    className="productImg custom-class-img"
    src="img/test"
  />
</div>
`;
